gremlin-core/pom.xml

```diff
@@ -146,6 +146,23 @@ limitations under the License.
                     </compilerArgs>
                 </configuration>
             </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-jar-plugin</artifactId>
+                <executions>
+                  <execution>
+                    <id>package-test-lib</id>
+                    <goals>
+                      <goal>test-jar</goal>
```

Comment:
we try to avoid test jars - `TestHelper`is already in `gremlin-test` - https://github.com/apache/tinkerpop/blob/06f5213a22986574751295042aaec3ce232b4cd2/gremlin-test/src/main/java/org/apache/tinkerpop/gremlin/TestHelper.java - yes, it is a small code duplication, but I'd prefer that to a special classified artifact.

Reply:
Resolved as extracting from gremlin-core/src/test/.../TestHelper  classes the     - CoreTestHelper class in gremlin-core/main/src/main/java/..../util where to keep the unique code to manage directories and files, inherited by gremlin-test/...TestHelper class, which is referred everywhere (but gremlin-core). This left in gremlin-core/src/test/.../TestHelper the only truly test method which is the only duplication kept. This way no new test artifact is needed at the cost of one simple method duplicated.